Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -435,30 +467,14 @@ open class RouteController: NSObject {
         BillingHandler.shared.beginBillingSession(for: .activeGuidance, uuid: sessionUUID)
 
         subscribeNotifications()
-        updateNavigator(with: routeProgress)
-        updateObservation(for: routeProgress)
+        updateNavigator(with: routeProgress, completion: {_ in })
```

Comment:
Technically speaking active guidance doesn't start until the completion is called. We can think how to improve this part.